app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,29 @@
-<h2>Log in</h2>
-
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
-  </div>
-
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "off" %>
-  </div>
-
-  <% if devise_mapping.rememberable? -%>
-    <div class="field">
-      <%= f.check_box :remember_me %>
-      <%= f.label :remember_me %>
+<div class="form-container">
+<%= devise_error_messages! %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h2>Log in</h2>
+      <%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { role: 'form'}) do |f| %>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :email %>
+        <%= f.email_field :email, autofocus: true, class: 'form-control input-lg' %>
+      </div>
+      <div class="form-group">
+        <%= f.label :password %>
+        <%= f.password_field :password, class: 'form-control input-lg' %>
+      </div>
+        <%= f.submit 'Log in', class: 'btn btn-default btn-lg' %>
+        <% if devise_mapping.rememberable? -%>
+          <div class="checkbox" style="width:150px">
```

Comment:
@cchanningallen Pulled it out completely because I don't see a need for it. Came with Devise or RailsLayout and I must have glazed over it. Thanks!